docs/source/Activities/Architecture_Review_Methodology.md

```diff
@@ -45,3 +45,14 @@ Included in the workbook will be a list of threat modeling findings and any addi
 ### Software Architecture Report
 
 The Architecture Review will identify the components of the system and how the security services are used in relation to those components.
+
+:::{figure-md} perform-architecture-review-process
```

Comment:
@JDziurlaj Two examples of how we could display the models. The javascript version is scrollable/zoomable, but not accessible and wouldn't be output in the PDF, which makes me think the figure version is the only one that will work.